tests/kernel/wave/wave_e2e_test.py

```diff
@@ -1,25 +1,35 @@
 import shark_turbine.kernel as tk
 import shark_turbine.kernel.lang as tkl
 import shark_turbine.kernel.wave as tkw
-from shark_turbine.kernel.wave.wave_sim import wave_sim
-from shark_turbine.kernel.lang.global_symbols import *
 import torch
 from numpy.testing import assert_allclose, assert_equal
 import pytest
 import sympy
 import os
 import torch
+import json
 
 _run_e2e = int(os.environ.get("WAVE_RUN_E2E_TESTS", 0))
-
 require_e2e = pytest.mark.skipif(not _run_e2e, reason="e2e tests are disabled")
+default_test_shapes = [(1, 128), (256, 64), (256, 128), (256, 256), (256, 1024)]
+
+user_specified_test_shapes = ""
+
+test_params_path = os.environ.get("TEST_PARAMS_PATH", None)

```

Comment:
(Saw this after a comment on https://github.com/iree-org/iree-turbine/pull/163#discussion_r1772383338)

Did you consider using pytest marks instead of environment variables and json files?

If all you want to enable is running different sets of tests in different workflows, pytest marks may be a better tool for the job than this configuration coming from files and environment variables.
* https://docs.pytest.org/en/stable/how-to/mark.html
* https://docs.pytest.org/en/stable/example/markers.html#mark-examples

With marks, you can define all the tests in the source file and then developers and automated workflows can choose to skip tests with a given mark (e.g. "large", "large_shapes", "benchmark", etc.).

This environment variable approach gives you more flexibility to customize the shapes without modifying the test source, which can help when running the tests from outside of this repository, but it doesn't look like that is a case you want to support at this moment. I think the ergonomics of setting an environment variable to a specific file path are quite poor for local development, especially if you plan on adding more of these files (one per test? each with its own environment variable?)